web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,29 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        e.stopPropagation();
+        this.clicks_specifics.push(e.target);
```

Comment:
e.target returns the DOM object. I can just create some object like `{ 'type': 'div', 'id': 'div_1' }` or something along those lines. Depends on what we want, but I think this is probably all we would need.